build.sbt

```diff
@@ -2,10 +2,46 @@ import Dependencies._
 
 lazy val `akkaserverless-java-sdk` = project
   .in(file("."))
-  .aggregate(sdk, testkit, tck, codegenCore, codegenJava, codegenJavaCompilationTest)
+  .aggregate(sdkCore, sdkJava, sdkScala, testkitJava, tck, codegenCore, codegenJava, codegenJavaCompilationTest)
 
-lazy val sdk = project
-  .in(file("sdk"))
+//FIXME duplicating most settings in sdkCore, sdkJava, sdkScala for now, cleanup later.
+lazy val sdkCore = project
+  .in(file("sdk/core"))
+  .enablePlugins(AkkaGrpcPlugin, BuildInfoPlugin, PublishSonatype)
+  .settings(
+    name := "akkaserverless-java-sdk-core",
```

Comment:
maybe `akkaserverless-jvm-sdk` or `akkaserverless-runtime`?

Reply:
I like the jvm-sdk suggestion